src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
+    private String color;
+
+    public Circle(double radius, String color) {
+        setColor(color);
+        this.radius = radius;
```

Comment:
According to the checklist, creating a figure should not pass expressions in the constructor. It's better to create a variable for the color argument before passing it to the constructor for better readability.